Solutions/Threat Intelligence/Package/createUiDefinition.json

```diff
@@ -216,529 +167,655 @@
               }
             ]
           },
           {
             "name": "analytic3",
             "type": "Microsoft.Common.Section",
-            "label": "TI map Domain entity to Web Session Events (ASIM Web Session schema)",
+            "label": "TI map Domain entity to DnsEvents",
             "elements": [
               {
                 "name": "analytic3-text",
                 "type": "Microsoft.Common.TextBlock",
                 "options": {
-                  "text": "This rule identifies Web Sessions for which the target URL hostname is a known IoC. This rule uses the [Advanced Security Information Model (ASIM)](https:/aka.ms/AboutASIM) and supports any web session source that complies with ASIM."
+                  "text": "Identifies a match in DnsEvents from any Domain IOC from TI"
                 }
               }
             ]
           },
           {
             "name": "analytic4",
             "type": "Microsoft.Common.Section",
-            "label": "TI map Domain entity to PaloAlto",
+            "label": "TI map Domain entity to EmailEvents",
             "elements": [
               {
                 "name": "analytic4-text",
                 "type": "Microsoft.Common.TextBlock",
                 "options": {
-                  "text": "Identifies a match in Palo Alto data in CommonSecurityLog table from any Domain IOC from TI"
+                  "text": "Identifies a match in EmailEvents table from any Domain IOC from TI"
                 }
               }
             ]
           },
           {
             "name": "analytic5",
             "type": "Microsoft.Common.Section",
-            "label": "TI map Domain entity to SecurityAlert",
+            "label": "TI map Domain entity to EmailUrlInfo",
             "elements": [
               {
                 "name": "analytic5-text",
                 "type": "Microsoft.Common.TextBlock",
                 "options": {
-                  "text": "Identifies a match in SecurityAlert table from any Domain IOC from TI"
+                  "text": "Identifies a match in EmailUrlInfo table from any Domain IOC from TI."
                 }
               }
             ]
           },
           {
             "name": "analytic6",
             "type": "Microsoft.Common.Section",
-            "label": "TI map Domain entity to Syslog",
+            "label": "TI map Domain entity to Web Session Events (ASIM Web Session schema)",
             "elements": [
               {
                 "name": "analytic6-text",
                 "type": "Microsoft.Common.TextBlock",
                 "options": {
-                  "text": "Identifies a match in Syslog table from any Domain IOC from TI"
+                  "text": "This rule identifies Web Sessions for which the target URL hostname is a known IoC. <br><br>This rule uses the [Advanced Security Information Model (ASIM)](https:/aka.ms/AboutASIM) and supports any web session source that complies with ASIM."
```

Comment:
Please remove these HTML tags <br><br>

Reply:
I removed the HTML tags from all their origins in the solution.